crimsobot/models/fun_facts.py

```diff
@@ -1,31 +1,79 @@
+import random
+from typing import Any
+
 from tortoise import fields
 from tortoise.models import Model
 
 from crimsobot.models import DiscordUser
 from crimsobot.models.user import User
 
 
+class NoFactsExist(Exception):
```

Comment:
I do not know if a custom exception is necessary, and if it is, I don't know if I've put this in the best place.